LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2024 Block, Inc.
```

Comment:
do we want this block or "the exchange authors" personally i dont care just known some do the latter